drawers/text_field.go

```diff
@@ -61,6 +59,35 @@ func NewTextFieldDrawer() *ElementDrawer {
 	}
 }
 
+func adjustTextField(text *elements.TextField) *elements.TextField {
+	fontName := text.Font.Name
+	res := *text
+
+	if fontName != "0" {
+		// For some reason width of DejavuSansMono needs to be scaled by 2
+		res.Font.Width *= 2
```

Comment:
Shouldn't this logic be part of `FontInfo`?

Reply:
As for 
> Wouldn't it be better to reverse this condition?
it should be for all ZPL fonts except 0 so it is essentially for fonts A-H. It will be a really long if I try to use ||